reconcile/quay_mirror.py

```diff
@@ -102,6 +103,13 @@ def process_sync_tasks(self):
         sync_tasks = defaultdict(list)
         for org, data in summary.items():
             for item in data:
+
+                if is_in_shard(item["name"]):
```

Comment:
what if we move this log statement to is_in_shard? that way we won’t need to repeat ourselves and we can omit the log if SHARDS is 1

Reply:
done